Editor/Window/ConnectToFleetInput.cs

```diff
@@ -111,6 +111,7 @@ private void OnSelectFleetDropdown(string fleetName)
                 _fleetIdText.text = currentFleet.FleetId;
                 _stateManager.AnywhereFleetName = currentFleet.Name;
                 _stateManager.AnywhereFleetId = currentFleet.FleetId;
+                _stateManager.AnywhereFleetLocation = _fleetManager.FleetLocation;
```

Comment:
I've added the same line of code to `OnAnywhereConnectClicked`, but see comment in GameLiftFleetManager about returning the location in the response instead of grabbing the constant